src/EditorFeatures/Core/Implementation/NavigationBar/NavigationBarController.cs

```diff
@@ -377,8 +379,6 @@ private void ProcessItemSelectionSynchronously(NavigationBarItem item, Cancellat
             if (item is NavigationBarProjectItem projectItem)
             {
                 projectItem.SwitchToContext();
-
-                // TODO: navigate to document / focus text view
```

Comment:
no clue waht this todo emans, and the behavior of switching the project context seems fine, so i removed.